src/housingpackage/ingest_data.py

```diff
@@ -0,0 +1,202 @@
+import argparse
+import pandas as pd
+import numpy as np
+import os
+import tarfile
+from six.moves import urllib
+from sklearn.model_selection import train_test_split
+from sklearn.model_selection import StratifiedShuffleSplit
+from sklearn.impute import SimpleImputer
+from logging import Logger
+
+from housingpackage.logger import configure_logger
+
+print("ingesting data")
+
+DOWNLOAD_ROOT = "https://raw.githubusercontent.com/ageron/handson-ml/master/"
+HOUSING_PATH = os.path.join("data/raw", "housing")
+HOUSING_URL = DOWNLOAD_ROOT + "datasets/housing/housing.tgz"
+imputer = SimpleImputer(strategy="median")
+
+
+def fetch_housing_data(housing_url=HOUSING_URL, housing_path=HOUSING_PATH):
```

Comment:
please add numpy style doc strings to all these functions so that they will go to the doc htmls
You can use VSCode docstring extension to setup template for each function